_posts/2017-06-03-numfocus-grant.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: NumFOCUS small grant for Python 3 support
+---
+
+We have generously been awarded a [small development grant][grant-post] by
+NumFOCUS to finally add
```

Comment:
"to finally add Py3 support"   is not really correct; we are actually "mostly" supporting Py 3 already. I would write to "fully support Py 3".